plugin/goto.py

```diff
@@ -40,22 +40,71 @@ def run(self, edit: sublime.Edit, event: 'Optional[dict]' = None) -> None:
                     request, lambda response: self.handle_response(response, pos))
 
     def handle_response(self, response: 'Optional[Any]', position: int) -> None:
+        def process_response_list(responses: list) -> List[Tuple[str, str, Tuple[int, int]]]:
```

Comment:
For instance, ST doesn't understand what this `List` class is, so you must wrap it in single quotes:

```
def process_response_list(responses: list) -> List[Tuple[str, str, Tuple[int, int]]]
```